### DISCUSSION TITLE
btc-rpc-explorer for StartOS

### DISCUSSION THREAD

**Discussion by aaeenneess** [janoside/btc-rpc-explorer#595] [General]
Hi there, I just want to ask if there is a possibility to make btc-rpc-explorer packaged for StartOS (https://start9.com/)? I recently moved my node from Umbrel on Rpi to StartOS on x86 platform, for better performance, and your great explorer is one thing that I miss a bit for detail insights about my node.

cheers and keep up the good work! :)

**Comment by pointbiz** [comment]
Have you tried to post an issue at their github?

I noticed that with Umbrel that's what has been done. Someone other than the repo owner requested that this project be added to Umbrel and it was. That method also worked for me to request Umbrel to package the newest release of this project.

**Reply by aaeenneess** [reply]
StartOS support told me that I can try to ask btc-rpc team to pack it. So Im just trying. ;)
